LibreHardwareMonitorLib/Hardware/Motherboard/SuperIOHardware.cs

```diff
@@ -1852,6 +1853,26 @@ private static void GetIteConfigurationsB(ISuperIO superIO, Manufacturer manufac
                         c.Add(new Control("CPU Optional Fan", 4));
                         break;
 
+                    case Model.X870_AORUS_ELITE_WIFI7: // ITE IT8696E
+                    case Model.X870E_AORUS_ELITE_WIFI7: // ITE IT8696E
+                        t.Add(new Temperature("System #1", 0));
+                        t.Add(new Temperature("PCH", 1));
+                        t.Add(new Temperature("CPU", 2));
+                        t.Add(new Temperature("PCIe x16", 3));
+                        t.Add(new Temperature("VRM MOS", 4));
+                        t.Add(new Temperature("EC_TEMP1", 5));
```

Comment:
Neither of these boards have an EC header. Where are you referencing these names/why is it included the way it is now?

Reply:
Please have a look at my [comment](https://github.com/LibreHardwareMonitor/LibreHardwareMonitor/pull/1510#issuecomment-2452782575) for an explanation on EC_TEMP1.

Reply:
Yeah, I was wrong about your specific board with EC Temp - I apologize for that.